test/controllers/answers_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AnswersControllerTest < ActionController::TestCase
+  # test "the truth" do
```

Comment:
It's a good time to start adding specs, before you get too far into development and it starts to seem too daunting. Look into using rspec, we can do a workshop on this in the next couple of weeks.